build.sbt

```diff
@@ -103,13 +103,16 @@ libraryDependencies ++= Seq(
     "org.julienrf" %% "play-json-derived-codecs" % "4.0.0",
     "org.json4s" %% "json4s-native" % json4sVersion,
     "org.json4s" %% "json4s-jackson" % json4sVersion,
-    "org.scalatest" %% "scalatest" % "3.0.5" % "test",
     "com.typesafe.play" %% "play-json-joda" % "2.6.9",
 
     "org.postgresql"           %  "postgresql"                   % "42.2.5",
     "org.scalikejdbc"          %% "scalikejdbc"                  % "3.1.0",
     "org.scalikejdbc"          %% "scalikejdbc-config"           % "3.1.0",
-    "org.scalikejdbc"          %% "scalikejdbc-play-initializer" % "2.6.0-scalikejdbc-3.1"
+    "org.scalikejdbc"          %% "scalikejdbc-play-initializer" % "2.6.0-scalikejdbc-3.1",
+
+    "org.scalatest" %% "scalatest" % "3.0.5" % "test",
+    "com.whisk" %% "docker-testkit-scalatest" % "0.9.9" % "test",
```

Comment:
😐 not a fan of launching containers this way.

Reply:
You said. Largely because it makes it hard to collaborate with other consumers using different CI or other reasons? This guy has an alternative approach: https://medium.com/@afsal.taj06/docker-compose-management-using-sbt-and-property-based-tests-bb8ae7df2759 happy to switch if it alleviates your concerns. I just don't want to think about spinning up, waiting for them to be ready and teaaring down containers.